galaxy/startup.sh

```diff
@@ -8,7 +8,7 @@ umount /var/lib/docker
 python ./export_user_files.py $PG_DATA_DIR_DEFAULT
 
 # Configure SLURM with runtime hostname.
-/home/galaxy/venv/bin/python /usr/sbin/configure_slurm.py
+python /usr/sbin/configure_slurm.py
```

Comment:
Included it in the SLURM recipe - https://github.com/galaxyproject/ansible-galaxy-extras/blob/master/tasks/slurm.yml. I am using the same pattern in planemo-machine.

Obviously it is not how most traditional clusters would manage SLURM so I created an issue to make this more of an optional thing (https://github.com/galaxyproject/ansible-galaxy-extras/issues/2) - but my thought is lets get something working for these two projects and then if there is time make the ansible roles generalize to more typical setups.